package.json

```diff
@@ -165,7 +165,7 @@
     "apollo-server-core": "^1.3.6",
     "apollo-server-errors": "^2.0.2",
     "apollo-server-hapi": "^1.3.6",
-    "archiver": "^3.1.1",
+    "archiver": "^5.2.0",
```

Comment:
should we upgrade `@types/archiver` too?

Reply:
good catch, I miss that. Updated